utils/networking.js

```diff
@@ -1,175 +1,74 @@
-    "polygon": 'https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-graph-matic',
-    "rinkeby": 'https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-graph-rinkeby'
-}
-
-//const sessionId = uuidv4();
-
-const DEFAULT_DATAS = {
-  "uri": undefined,
-}
-
-const DEFAULT_URI_CONTENT = {
-  "name": "Default banner",
-  "description": "This is the default banner that would be displayed ipsum",
-  "image": "https://ipfs.zesty.market/ipfs/QmWBNfP8roDrwz3XQo4qpu9fMxvUSTn8LB7d4JK7ybrfZ2/assets/zesty-ad-square.png",
-  "url": "https://www.zesty.market"
-}
+const BEACON_API_BASE = 'https://beacon.zesty.market'
+const BEACON_GRAPHQL_URI = 'https://beacon2.zesty.market/zgraphql'
 
-/**
- * Queries The Graph to retrieve NFT information for the space.
- * @param {string} space The space ID
- * @param {string} creator The wallet address of the creator
- * @param {string} network The network to post metrics to
- * @returns An object with the requested space information, or a default if it cannot be retrieved.
- */
-const fetchNFT = async (space, network = 'polygon') => {
-  const currentTime = Math.floor(Date.now() / 1000);
-  return axios.post(ENDPOINTS[network], {
-    query: `
-      query {
-        tokenDatas (
-          where: {
-            id: "${space}"
-          }
-        )
-        { 
-          sellerNFTSetting {
-            sellerAuctions (
-              first: 5
-              where: {
-                contractTimeStart_lte: ${currentTime}
-                contractTimeEnd_gte: ${currentTime}
-                cancelled: false
-              }
-            ) {
-              id
-              buyerCampaigns {
-                id
-                uri
-              }
-              buyerCampaignsApproved
-              buyerCampaignsIdList
-            }
-          }
-          id
-        }
-      }
-    `
-  })
-  .then((res) => {
-    return parseGraphResponse(res);
-  })
-  .catch((err) => {
-    console.log(err);
-    return DEFAULT_DATAS;
-  })
-};
-
-/**
- * Parses the response from The Graph to find the latest auction campaign.
- * @param {Object} res The response object from The Graph.
- * @returns An object containing either the latest auction campaign or default data.
- */
-const parseGraphResponse = res => {
-  if (res.status != 200) {
-    return DEFAULT_DATAS 
-  }
-  let sellerAuctions = res.data.data.tokenDatas[0]?.sellerNFTSetting?.sellerAuctions;
-  let latestAuction = null;
-  sellerAuctions?.[0]?.buyerCampaignsApproved?.find((campaign, i) => {
-    if (campaign) {
-      const campaignId = sellerAuctions[0].buyerCampaignsIdList[i]; // Graph stores as string, coerce to int
-      latestAuction = sellerAuctions[0].buyerCampaigns.find(campaign => campaign.id === campaignId)
-    }
-  });
-  
-  if (latestAuction == null) {
-    return DEFAULT_DATAS 
-  }
+const DB_ENDPOINT = 'https://api.zesty.market/api';
+// TODO: Determine best way to enable switching to staging
```

Comment:
I would recommend using a flag that gets toggled through an environment variable on the machine.

Reply:
Will handle enabling this for forwarding+SDK in the future